src/Modules/UI/Dnn.EditBar.UI/editBar/resources/ContentEditorManager/Styles/ContentEditor.css

```diff
@@ -39,6 +39,32 @@
 .dnnModuleManager .addModuleHandler.active span {
     /*display: inline-block;*/
 }
+.dnnModuleManager .dnnPaneLabel {
+    display: inline-block;
+    box-sizing: border-box;
+    position: absolute;
+    left: 0px;
+    top: 0px;
+    height: 16px;
+    line-height: 12px;
+    color: #333;
+    font-size: 9px;
+    background-color: #fff;
+    padding: 2px 4px;
+    text-transform: uppercase;
+    font-weight: bold;
+    -webkit-touch-callout: none;
+    -webkit-user-select: none;
+    -khtml-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    transition-duration: 200ms;
+    opacity: 0;
+}
+.dnnModuleManager.active-pane > .dnnPaneLabel {
+    opacity: 1;
```

Comment:
Is it using opacity to control visibility?

Reply:
Yes, it will display at top left of the pane.